test/basic.html

```diff
@@ -469,7 +516,9 @@
 
         // Wait one tick for observeNodes.
         Polymer.Base.async(function() {
-          form._form.querySelector('input[type=submit]').click();
+          form.$.content.assignedNodes()
+            .find((n) => n.nodeType === Node.ELEMENT_NODE)
```

Comment:
Makes sense, will update everything for ES5. Honestly I wasn't sure if you'd consider merging my updates and my use-case environment supports ES6.